rotkehlchen/externalapis/covalent.py

```diff
@@ -25,7 +26,11 @@
 CONST_RETRY = 1
 DATE_FORMAT_COVALENT = '%Y-%m-%dT%H:%M:%SZ'
 DEFAULT_API = 'covalent'
-KEY = 'ckey_cb70cd52c833489ea522f104e1a'
+COVALENT_KEYS = (
+    'ckey_cb70cd52c833489ea522f104e1a',
```

Comment:
shouldn't the used up key be removed? Otherwise at 1/3 of the cases we go the same problem.

Also please make sure to not use it in the CI at all. Or use different set of keys for testing. To be totally safe you can do 3 keys in __debug__ and 3 keys in production

Ofc as discussed this is just a temporary solution